app/index.html

```diff
@@ -273,23 +287,28 @@ <h2>
       
       <div class="newsletter__box">
         <div class="newsletter__left">
-          <h1>
+          <h1 class="title-primary">
             Subscribe Newsletter
           </h1>
-          <p>
+
+          <p class="content">
             I will update good news and promotion service not spam
           </p>
         </div>
 
         <div class="newsletter__right">
           <div class="newsletter__right-contact">
-            <input type="text" placeholder="Enter your email address..">
-            <a href="#" class="newsletter__right-contact-btn btn">Contact Now</a>
+            <input type="text" placeholder="Enter your email address.." class="newsletter__right-contact--input">
```

Comment:
Tu jak wyżej, błędnie stosowany jest `--` z bema. Więc raczej `newsletter__right-contact-input`
Przykład gdzie zasadny jest `--`:

```
<div class="info">
    <h3 class="info__title">Normal Title</h3>
    <h3 class="info__title info__title--danger">Danger Title</h3>
</div>

<style>
    .info {
        display: flex;
    }

    .info__title {
        font-size: 24px;
        font-weight: 600;
        color: blue;
    }

    .info__title--danger {
        color: red;
        font-weight: 800;
    }
</style>

```

lub to samo z scss zagniezdzeniem:

```
 .info {
        display: flex;

        &__title {
            font-size: 24px;
            font-weight: 600;
            color: blue;

            &--danger {
                color: red;
                font-weight: 800;
            }
        }
    }
```